var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -177,15 +177,14 @@ def build_voronoi(self):
                         'compute_voronoi_atom.h')
 
     def build(self, spec, prefix):
-        for pkg in self.supported_packages:
```

Comment:
I think you also change where this is called from, before it was from the root folder and then `with working_dir('lib/atc'):`, now it's from ` with working_dir('src'):` and then `with working_dir('lib/atc'):`.